README.md

```diff
@@ -12,7 +12,7 @@ The package can be installed by adding `membrane_mp4_plugin` to your list of dep
 ```elixir
 defp deps do
   [
-    {:membrane_mp4_plugin, "~> 0.32.0"}
+    {:membrane_mp4_plugin, "~> 0.33.0"}
```

Comment:
I think it can be a patch bump

Reply:
As discussed with PMs, this changes the structure of files that people could depend on so minor version stays.